### DISCUSSION TITLE
Incorrect API setup

### DISCUSSION THREAD

**Discussion by cbrown297** [rocksdanister/lively#1124] [General]
When going to use the puddles wallpaper that shows the weather it displays an error message. The error message reads "please enter openweathermap.org/api/city  information correctly. " I'm sure it's just a typo somewhere or the function for the city isn't being read:)

**Comment by rocksdanister** [comment]
When you right-click the wallpaper you will get an option to customise:
![image](https://user-images.githubusercontent.com/17554161/158065813-60d6d440-57c2-4884-b75d-724c6ad54794.png)

From there you can enter the openweathermap key and the city name; openweathermap key you will have to create (google search: how to create openweathermap api key.)

The key will take some time to activate so try after sometime once you create it.
